Library/ViewModels/BackerDashboardViewModelTests.swift

```diff
@@ -96,6 +96,18 @@ internal final class BackerDashboardViewModelTests: TestCase {
       // Signals that emit just once because they rely on the datasource tab index to exist first.
       self.pinSelectedIndicatorToTab.assertValues([.backed])
       self.pinSelectedIndicatorToTabAnimated.assertValues([false])
+
+      let user2 = user
+        |> \.name .~ "Updated user"
+        |> \.stats.starredProjectsCount .~ 60
+
+      withEnvironment(apiService: MockService(fetchUserSelfResponse: user2)) {
+        self.vm.inputs.projectSaved()
```

Comment:
Do you think this should maybe go in its own test? Something like `testProjectSavedRefreshesData`.

Reply:
I thought about it but then decided to follow the convention of the file (what I don't like at all, because it makes the tests too big and hard to see everything that's being tested)... But after a second thought I think I should do a separated test.